src/app/[locale]/rest/[[...params]]/page.tsx

```diff
@@ -66,6 +67,7 @@ export default function RequestPage() {
       <div className="min-h-[200px]">
         {response && <CustomResponse response={response} />}
       </div>
+      <GeneratedCodeSection />
```

Comment:
чтоб респонс был в самом низу

```suggestion
 <GeneratedCodeSection />
      <div className="min-h-[200px]">
        {response && <CustomResponse response={response} />}
      </div>
     
```